scap/scc/scan.sls

```diff
@@ -9,6 +9,15 @@ create scc output directory:
     - name: '{{ scc.output_dir }}'
     - makedirs: True
 
+set permissions option:
+  file.replace:
+    - name: 'C:\\Program Files\\SCAP Compliance Checker {{ scc.version }}\\options.xml'
```

Comment:
This state supports linux and windows. Need to gate this to the platform somehow, or otherwise check for the option in the linux config file (if it exists)

Reply:
I'll check linux.  I'll add the gating if needed.

Reply:
Linux has the same config file and parameter.  However, initial testing on EL7 indicates that this parameter (also set to 1) is not affecting the functionality in Linux.  Will gate this change to Windows only.